client/my-sites/themes/auto-loading-homepage-modal.jsx

```diff
@@ -96,32 +108,74 @@ class AutoLoadingHomepageModal extends Component {
 				buttons={ [
 					{
 						action: 'keepCurrentTheme',
-						label: translate( 'No, keep my current theme' ),
+						label:
+							show_on_front === 'posts'
+								? translate( 'Keep my current theme' )
+								: translate( 'No, keep my current theme' ),
 						isPrimary: false,
 						onClick: this.closeModalHandler( false ),
 					},
 					{
 						action: 'activeTheme',
-						label: translate( 'Yes, activate %(themeName)s', {
-							args: { themeName },
-						} ),
+						label:
+							show_on_front === 'posts'
+								? translate( 'Activate %(themeName)s', { args: { themeName } } )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Yes, Activate %(themeName)s' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/switch-theme-homepage-flow)